public/app/core/components/PermissionList/DisabledPermissionListItem.tsx

```diff
@@ -28,7 +27,7 @@ export default class DisabledPermissionListItem extends Component<Props, any> {
             <Select
               options={dashboardPermissionLevels}
               onChange={() => {}}
-              isDisabled={true}
+              disabled={true}
               className="gf-form-select-box__control--menu-right"
```

Comment:
Could we maybe also get rid of this class here? Doesn't seems like it has any effect.